rotkehlchen/db/addressbook.py

```diff
@@ -111,10 +124,11 @@ def add_addressbook_entries(
                             (entry.address,),
                         )
 
-                    write_cursor.execute(
-                        'INSERT INTO address_book (address, name, blockchain) VALUES (?, ?, ?)',
-                        entry.serialize_for_db(),
-                    )
+                    query = 'INSERT '
+                    if exists_ok is True:
```

Comment:
This is yet another hack. Maaaaan wtf?

1. If you wanted to do this move the entire thing out of the for loop and call it `query_prefix`. It's the same for all entries in the loop.
2. I guess this is only needed for you now since the logic you use in curve pools deletes everything and rewrites it all over again. Every. Single. Time. It runs. Doesn't this look like a problem to you? And instead of adding this hack here perhaps find a way for it not to happen instead and not have to go through so much logic for nothing?